var/spack/repos/builtin/packages/miopen-hip/package.py

```diff
@@ -15,6 +15,7 @@ class MiopenHip(CMakePackage):
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('4.1.0', sha256='068b1bc33f90fe21d3aab5697d2b3b7b930e613c54d6c5ee820768579b2b41ee')
```

Comment:
I just noticed the versions for this package are tags, not releases.  The recommendation in such cases is to use the `commit` (i.e., `commit=<hash>`) instead of the `sha256`.

There's some documentation on this at https://spack.readthedocs.io/en/latest/packaging_guide.html?highlight=version%20commit#git.